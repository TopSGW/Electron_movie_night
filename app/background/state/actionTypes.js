'use strict'

export const ADD_MOVIE_FILE = 'add-movie-file'
export const CRAWL_COMPLETE = 'crawl-complete'
export const CRAWL_START = 'crawl-start'
export const MOVIE_FILE_COMPLETE = 'movie-file-complete'
export const MOVIE_FILE_ERROR = 'movie-file-error'
