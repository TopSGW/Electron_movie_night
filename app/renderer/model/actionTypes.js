'use strict'

// Crawl actions
export const CRAWL_DIRECTORY = 'crawl-directory'
export const IS_CRAWLING = 'is-crawling'
export const SHOW_CRAWL_STATS = 'show-crawl-state'
export const SET_CRAWL_STATS = 'set-crawl-stats'

// Database loaded actions
export const DATABASE_LOADED = 'database-loaded'

// Featured movie actions
export const CLEAR_FEATURED_MOVIE = 'clear-featured-movie'
export const UPDATE_FEATURED_MOVIE = 'update-featured-movie'

// Movie database actions
export const UPDATE_MOVIE_DATABASE = 'update-movie-database'

// Search actions
export const CLEAR_SEARCH_RESULTS = 'clear-search-results'
export const CLEAR_SEARCH_QUERY = 'clear-search-query'
export const UPDATE_SEARCH_CATEGORY = 'update-search-category'
export const UPDATE_SEARCH_QUERY = 'update-search-query'
