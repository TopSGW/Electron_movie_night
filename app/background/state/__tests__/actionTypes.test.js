'use strict'
/* globals test, expect */

import * as actionTypes from '../actionTypes'

test('action types have not change', () => {
  expect(actionTypes).toMatchSnapshot()
})
