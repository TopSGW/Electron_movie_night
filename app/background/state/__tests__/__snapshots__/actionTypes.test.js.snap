// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`action types have not change 1`] = `
Object {
  "ADD_MOVIE_FILE": "add-movie-file",
  "CRAWL_COMPLETE": "crawl-complete",
  "CRAWL_START": "crawl-start",
  "MOVIE_FILE_COMPLETE": "movie-file-complete",
  "MOVIE_FILE_ERROR": "movie-file-error",
}
`;
